File: estudo/neri/algoritmo/c/Consorcio.c
#include <stdio.h>
int main(){
    float total_prest, qtd_prest_pagas, valor_prest, total_prest_pagas, saldo_devedor;
    printf("Informe a quantidade de parcelas: ");
    scanf("%f",&total_prest);
    printf("Informe a quantidade de parcelas pagas: ");
    scanf("%f",&qtd_prest_pagas);
    printf("Informe o valor das parcelas: ");
    scanf("%f",&valor_prest);
    total_prest_pagas = valor_prest * qtd_prest_pagas;
    saldo_devedor = valor_prest * (total_prest - qtd_prest_pagas);
    printf("\n O total de prestacoes pagas foi de: = %f",total_prest_pagas);
    printf("\n O saldo devedor ficou de: = %f",saldo_devedor);
}
   

File: estudo/neri/algoritmo/java/Consorcio.java
import javax.swing.JOptionPane;

public class Consorcio {
    public static void main(String args[]){
        float total_prest, qtd_prest_pagas, valor_prest, total_prest_pagas, saldo_devedor;
        total_prest = Float.parseFloat(JOptionPane.showInputDialog("Informe a quantidade de prestacoes: "));
        qtd_prest_pagas = Float.parseFloat(JOptionPane.showInputDialog("Informe a quantidade de parcelas pagas: "));
        valor_prest = Float.parseFloat(JOptionPane.showInputDialog("Informe o valor da parcela: "));
        total_prest_pagas = valor_prest * qtd_prest_pagas;
        saldo_devedor = valor_prest * (total_prest - qtd_prest_pagas);
        JOptionPane.showMessageDialog(null,"\n O total de prestacoes foi de: = "+total_prest
        +"\n Foram pagas: = "+qtd_prest_pagas+"\n O valor da prestacao e de: = "+valor_prest
        +"\n O total de prestacoes pagas foi de: = "+total_prest_pagas+"\n O saldo devedor ficou de: = "+saldo_devedor);
    }
}
